src/components/autosuggest/autosuggest.spec.js

```diff
@@ -338,9 +363,19 @@ describe('script: autosuggest', () => {
 
       await page.type('.ons-js-autosuggest-input', 'Eng', { delay: 20 });
 
-      const suggestionCount = await page.$$eval('.ons-autosuggest__option', nodes => nodes.length);
+      const suggestionCount = await page.$$eval('.ons-autosuggest__option', (nodes) => nodes.length);
       expect(suggestionCount).toBe(1);
     });
+
+    it('gets the language if set', async () => {
+      await setTestPage('/test', renderComponent('autosuggest', EXAMPLE_AUTOSUGGEST_WITH_LANGUAGE));
+
+      await page.type('.ons-js-autosuggest-input', 'Eng', { delay: 20 });
+
+      const attributes = await getNodeAttributes(page, '.ons-js-autosuggest');
+      const languageCount = attributes['data-lang'];
+      expect(languageCount).toBeDefined;
```

Comment:
Think we should check that this then equals what it is set to rather than just that its been defined